.travis.yml

```diff
@@ -22,10 +26,11 @@ before_install:
   - conda update -q conda
 
 install:
-  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION pandas=$PANDAS_VERSION scipy=$SCIPY_VERSION libgfortran=$LIBGFORTRAN_VERSION
+  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION pandas=$PANDAS_VERSION scipy=$SCIPY_VERSION libgfortran=3.0
   - source activate testenv
-  - CACHE_DIR="$HOME/.cache/.pip/pip_np""$CONDA_NPY"
+  - CACHE_DIR="$HOME/.cache/.pip/pip_np"
```

Comment:
Where did we end up on a pip cache per numpy version? I think it was useful in zipline, and here we're still naming it `_np`, but seems like we're not using it for that purpose here.

Reply:
removed the cache dir

Reply:
@vikram-narayan I'm not sure, but it seemed like @richafrank was asking if we wanted to have a separate cache dir per numpy version. Is there a reason we think we don't want any caching here if we had it for zipline?

Reply:
earlier Rich had said that the cache dir is specified in zipline so that libs compiled against numpy's C API don't stomp on each other. That doesn't appear to be happening on travis, so I don't _think_ it's necessary here, though I'm not super knowledgeable about how dependencies work on travis/appveyor. fwiw empyrical's build matrix also has different versions of pandas/numpy and doesn't use separate cache dirs per numpy version

Reply:
Gotcha. Yeah, I don't think we need separate cache dirs for different numpy builds, but I think we do want to cache pip downloads between builds so that we don't have to re-download them every time. It might be that travis already does that by default though?

Reply:
From https://docs.travis-ci.com/user/caching/#pip-cache, it looks like we can just do `cache: pip` and that should handle our needs here?

Reply:
fixed